target/scripts/build/packages.sh

```diff
@@ -130,7 +130,7 @@ function _install_fail2ban
   curl -Lkso fail2ban.deb "${FAIL2BAN_DEB_URL}"
   curl -Lkso fail2ban.deb.asc "${FAIL2BAN_DEB_ASC_URL}"
 
-  FINGERPRINT=$(LANG=C gpg --verify fail2ban.deb.asc fail2ban.deb 2>&1 | sed -n 's#Primary key fingerprint: \(.*\)#\1#p')
+  FINGERPRINT=$(LANG=C gpg --verify fail2ban.deb.asc fail2ban.deb |& sed -n 's#Primary key fingerprint: \(.*\)#\1#p')
```

Comment:
I didn't expect `|&` to have much luck with search engines, but [I stand corrected](https://stackoverflow.com/questions/35384999/what-does-mean-in-bash) :sweat_smile: 

I suppose we have a lot of bash specific syntax throughout the project that a shell like `ash` probably isn't compatible?

Reply:
I don't know exactly, how compatible ash/dash etc. are to bash. But as we wrote our scripts for bash, I am very fine with using bash specific syntax to unleash the full power 😆

Reply:
@docker-mailserver/maintainers While we at it: what about replacing the sed command with:

`grep -oP '(?<=Primary key fingerprint: ).+'`

Regex explanation: The `?<=` within the group means --> ignore this group and match only `.+` that comes after.

This seems simpler and easier read (for me at least).

But no need for a lengthy discussion IMO. Just give a 👎 to keep it as it is, or 👍 to go with `grep`.

Reply:
I don't think it's worth it personally. Very minor improvement, and not something I'm going to remember, or like looking up if I saw it months later and forgot (`git blame` aside).

The `sed` command is familiar enough, I've seen the usage here elsewhere in the project, so it's easier to maintain.

Reply:
I do not see an inherent need for this change as well - doesn't really matter I think. I'd just like to merge this sometime soon so we can get a feature freeze in place for v11.2.0 :)